core/arch/arm/kernel/link.mk

```diff
@@ -30,6 +30,7 @@ entries-unpaged += sm_init
 entries-unpaged += core_init_mmu_regs
 entries-unpaged += sem_cpu_sync
 entries-unpaged += generic_boot_get_handlers
+entries-unpaged += plat_cpu_reset_late
```

Comment:
Use the `KEEP_PAGER()` macro instead please. A good spot is probably somewhere in `core/arch/arm/kernel/generic_boot.c`